endpoints/crab.py

```diff
@@ -21,16 +21,16 @@ def remove(response):
         t = text.upper().replace(', ', ',').split(',')
         clip = VideoFileClip("assets/crab/template.mp4")
 
-        text = TextClip(t[0], fontsize=48, color='white', font='Verdana')
+        text = TextClip(t[0] + '🅱️', fontsize=48, color='white', font='Verdana')
```

Comment:
I think we should display some placeholder text or reject the request if the provided arguments don't meet the requirements.

Reply:
Reject please, no need to waste cpu